spec/operators/windowCount-spec.js

```diff
@@ -3,31 +3,103 @@ var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
 
 describe('Observable.prototype.windowCount', function () {
-  function mergeMapfunction(x) {
-    return x.toArray();
-  }
+  it('should emit windows with count 2 and skip 1', function () {
+    var source = hot('^-a--b--c--d--|');
+    var subs =       '^             !';
+    var expected =   'u-v--x--y--z--|';
+    var u = cold(    '--a--(b|)      ');
+    var v = cold(      '---b--(c|)   ');
+    var x = cold(         '---c--(d|)');
+    var y = cold(            '---d--|');
+    var z = cold(               '---|');
```

Comment:
The problem is also that I don't think we are able to have both "no empty window in the end" and "first window starts immediately". Try to fulfill one and the other is not met.